po/en_GB.po

```diff
@@ -0,0 +1,156 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the io.github.lainsce.Colorway package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+# Will Thompson <wjt@endlessos.org>, 2023.
+#
+#. TRANSLATORS: 'Name <email@domain.com>' or 'Name https://website.example'
+#: src/window.vala:591
```

Comment:
I just want to highlight something I also included in the commit message.

Note that marking the empty string for translation in window.vala as
follows:

            // TRANSLATORS: 'Name <email@domain.com>' or 'Name https://website.example/'
            string translators = (_(""));

triggers a warning when generating the template, and based on the
message presumably does not work as intended at runtime:

    src/window.vala:591: warning: Empty msgid.  It is reserved by GNU gettext:
                                  gettext("") returns the header entry with
                                  meta information, not the empty string.

Reply:
It works with the UI library being used, don't worry.